Library/Formula/megatools.rb

```diff
@@ -27,7 +28,7 @@ def install
   test do
     # Downloads a publicly hosted file and verifies its contents.
     system "#{bin}/megadl",
-      "https://mega.co.nz/#!3Q5CnDCb!PivMgZPyf6aFnCxJhgFLX1h9uUTy9ehoGrEcAkGZSaI",
+      "https://mega.nz/#!3Q5CnDCb!PivMgZPyf6aFnCxJhgFLX1h9uUTy9ehoGrEcAkGZSaI",
```

Comment:
Please submit this chunk upstream to `Homebrew/homebrew`. See [Contributing to Linuxbrew](https://github.com/Linuxbrew/linuxbrew/blob/master/.github/CONTRIBUTING.md#contributing-to-linuxbrew)

> Patches to fix issues that you have reproduced on both Linuxbrew and Homebrew should be sent to Homebrew.